keras/src/backend/numpy/math.py

```diff
@@ -144,6 +144,15 @@ def fft2(x):
     return np.array(real), np.array(imag)
 
 
+def ifft2(x):
```

Comment:
Please use the built-in version. https://numpy.org/doc/stable/reference/generated/numpy.fft.ifft2.html

Reply:
updated to use built-in version